pkg/ui/workspaces/db-console/BUILD.bazel

```diff
@@ -12,157 +14,317 @@ exports_files(
-    "@npm_db_console//@types/react",
-    "@npm_db_console//@types/react-copy-to-clipboard",
-    "@npm_db_console//@types/react-dom",
-    "@npm_db_console//@types/react-helmet",
-    "@npm_db_console//@types/react-paginate",
-    "@npm_db_console//@types/react-redux",
-    "@npm_db_console//@types/react-router-dom",
-    "@npm_db_console//@types/react-select",
-    "@npm_db_console//@types/redux",
-    "@npm_db_console//@types/redux-saga",
-    "@npm_db_console//@types/redux-thunk",
-    "@npm_db_console//@types/sinon",
-    "@npm_db_console//@types/storybook__react",
-    "@npm_db_console//@typescript-eslint/eslint-plugin",
-    "@npm_db_console//@typescript-eslint/parser",
-    "@npm_db_console//babel-loader",
-    "@npm_db_console//babel-plugin-import",
-    "@npm_db_console//bytebuffer",
-    "@npm_db_console//cache-loader",
-    "@npm_db_console//chai",
-    "@npm_db_console//chalk",
-    "@npm_db_console//copy-webpack-plugin",
-    "@npm_db_console//css-loader",
-    "@npm_db_console//d3",
-    "@npm_db_console//d3-geo-projection",
-    "@npm_db_console//enzyme",
-    "@npm_db_console//enzyme-adapter-react-16",
-    "@npm_db_console//escodegen",
-    "@npm_db_console//eslint",
-    "@npm_db_console//eslint-plugin-prettier",
-    "@npm_db_console//eslint-plugin-react",
-    "@npm_db_console//eslint-plugin-react-hooks",
-    "@npm_db_console//esbuild-loader",
-    "@npm_db_console//espree",
-    "@npm_db_console//estraverse",
-    "@npm_db_console//express",
-    "@npm_db_console//fetch-mock",
-    "@npm_db_console//file-loader",
-    "@npm_db_console//fork-ts-checker-webpack-plugin",
-    "@npm_db_console//glob",
-    "@npm_db_console//html-webpack-template",
-    "@npm_db_console//http-proxy-middleware",
-    "@npm_db_console//jsdoc",
-    "@npm_db_console//less-loader",
-    "@npm_db_console//minimist",
-    "@npm_db_console//mocha",
-    "@npm_db_console//nib",
-    "@npm_db_console//prettier",
-    "@npm_db_console//prop-types",
-    "@npm_db_console//protobufjs",
-    "@npm_db_console//redux-saga-test-plan",
-    "@npm_db_console//rimraf",
-    "@npm_db_console//semver",
-    "@npm_db_console//sinon",
-    "@npm_db_console//source-map-loader",
-    "@npm_db_console//string-replace-webpack-plugin",
-    "@npm_db_console//moment-timezone-data-webpack-plugin",
-    "@npm_db_console//style-loader",
-    "@npm_db_console//stylint",
-    "@npm_db_console//stylus",
-    "@npm_db_console//stylus-loader",
-    "@npm_db_console//thread-loader",
-    "@npm_db_console//tmp",
-    "@npm_db_console//topojson",
-    "@npm_db_console//ts-loader",
-    "@npm_db_console//uglify-js",
-    "@npm_db_console//uplot",
-    "@npm_db_console//url-loader",
-    "@npm_db_console//us-atlas",
-    "@npm_db_console//webpack",
-    "@npm_db_console//webpack-cli",
-    "@npm_db_console//webpack-dev-server",
-    "@npm_db_console//webpack-visualizer-plugin",
-    "@npm_db_console//webpackbar",
-    "@npm_db_console//world-atlas",
+    ":node_modules/@babel/core",
```

Comment:
Any particular reason we included `:node_modules/` here but used a list comprehension in `pkg/ui/workspaces/cluster-ui/BUILD.bazel`?

Reply:
No reason in particular. I updated it so they look similar.